crates/iota-sdk-ffi/src/types/graphql.rs

```diff
@@ -476,30 +570,223 @@ pub enum TransactionBlockKindInput {
     EndOfEpochTx,
 }
 
-#[uniffi::remote(Record)]
+#[derive(Clone, Debug, uniffi::Record)]
+pub struct BigInt {
+    pub value: String,
+}
+
+impl From<iota_graphql_client::query_types::BigInt> for BigInt {
+    fn from(value: iota_graphql_client::query_types::BigInt) -> Self {
+        BigInt { value: value.0 }
+    }
+}
+
+impl From<BigInt> for iota_graphql_client::query_types::BigInt {
+    fn from(value: BigInt) -> Self {
+        iota_graphql_client::query_types::BigInt(value.value)
+    }
+}
+
+#[derive(Clone, Debug, uniffi::Record)]
+pub struct DateTime {
+    pub value: String,
+}
+
+impl From<iota_graphql_client::query_types::DateTime> for DateTime {
+    fn from(value: iota_graphql_client::query_types::DateTime) -> Self {
+        DateTime { value: value.0 }
+    }
+}
+
+impl From<DateTime> for iota_graphql_client::query_types::DateTime {
+    fn from(value: DateTime) -> Self {
+        iota_graphql_client::query_types::DateTime(value.value)
+    }
+}
+
+/// Information about pagination in a connection.
+#[derive(Clone, Debug, uniffi::Record)]
 pub struct PageInfo {
```

Comment:
We should revert this, since this can be remotely derived

Reply:
With `#[uniffi::remote(Record)]` or how?

Reply:
https://github.com/iotaledger/iota-rust-sdk/pull/71/commits/c9c2d13a26bf593764b57d4cc1118c58e42bc3f3